language/tools/cost-synthesis/src/lib.rs

```diff
@@ -9,4 +9,3 @@ pub mod global_state;
 pub mod module_generator;
 pub mod natives;
 pub mod stack_generator;
-pub mod vm_runner;
```

Comment:
@tzakian vm_runner contained a macro that was only used from `main` and was giving me a ton of grief.
So I removed it and inlined the code in main. Take a look there for more comments